tests/scripts/expect/cli-diags.exp

```diff
@@ -153,6 +167,9 @@ send "diag repeat 1\n"
 expect "length 0x7f"
 expect "Done"
 
+send "diag repeat stop\n"
+expect "Done"
```

Comment:
Why do we add these two lines here? It seems that previously the script can work without the 2 lines.

Reply:
Because if we don't run the command `diag repeat stop`, the command `diag repeat` and `diag send` will run at the same time in the later steps. It will cause the command `diag send` get stuck. I think we should push a new CL to forbid the diag module to run the command `diag repeat` and `diag send` will run at the same time.